tvm/src/codegen/hlsc/codegen_shls.cc

```diff
@@ -795,14 +795,15 @@ void CodeGenStratusHLS::AddFunction(
 }
 
 void CodeGenStratusHLS::PrintType(Type t, std::ostream& os) {
-  PrintType(t, os, true);
+  PrintType(t, os, false);
 }
 
 void CodeGenStratusHLS::PrintType(Type t, std::ostream& os, bool is_index) {
   bool big = t.bits() > 64;
   if (big && is_index) {
```

Comment:
so this condition is always false in SHLS?

Reply:
Stratus HLS doesn't allow big int type array index, so there's this condition. I think I should change that to default false.

Reply:
Oh I shouldn't change to default false. 

`PrintType` is inherited from VHLS, we have to override this function and call `PrintType(Type t, std::ostream& os, bool is_index). Otherwise it falsely prints out `ap_int`. Let me add some comment to illustrate this.